main.tf

```diff
@@ -0,0 +1,123 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+  backend "azurerm" {
+    resource_group_name  = "Cohort28_KinEbr_ProjectExercise"
+    storage_account_name = "kingsleymod12store"
+    container_name       = "kingsleymod12storecontainer"
+    key                  = "terraform.tfstate"
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_resource_group" "main" {
+  name = "Cohort28_KinEbr_ProjectExercise"
+}
+
+resource "azurerm_service_plan" "main" {
+  name                = "terraformed-asp"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  os_type             = "Linux"
+  sku_name            = "B1"
+}
+
+resource "azurerm_linux_web_app" "main" {
+  name                = "kingsleymod12app"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  service_plan_id     = azurerm_service_plan.main.id
+
+  site_config {
+    always_on  = false
+    ftps_state = "FtpsOnly"
+    http2_enabled = true
+
+    application_stack {
+      docker_image     = "kingbrookey/my-todo-app-mod8"
+      docker_image_tag = "latest"
+    }
+  }
+
+  app_settings = {
+    "API_KEY"                          = var.API_KEY
+    "API_TOKEN"                        = var.API_TOKEN
```

Comment:
The fact these are all required suggests we're still using an old (pre-mongo) version of your application, is that intentional?